index.js

```diff
@@ -55,7 +55,13 @@ function main() {
           // Run goldbach here
           let max_int = Number(processNum.trim())
           gbf.set_max_number(max_int)
```

Comment:
This was missing a semi-colon
```suggestion
          gbf.set_max_number(max_int);
```